__tests__/invoices/helpers.ts

```diff
@@ -0,0 +1,77 @@
+import { LnInvoice } from "@app/graphql/generated"
```

Comment:
not sure having the helper.ts/test fixtures separate from the tests to be useful. I would suggest to put them in the same file?

Reply:
I can move them to be in the same file.